Real_world_examples/Scalable_machine_learning/1_Extract_training_data.ipynb

```diff
@@ -127,7 +127,7 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    input_data_sample = input_data.sample(150).reset_index(drop=True)
</pre><p><span class="mention" data-value="cbur24" data-denotation-char="@">﻿<span contenteditable="false">@cbur24</span>﻿</span> This uses 150 but the comment says 100</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>